lib/chromedriver.js

```diff
@@ -625,11 +625,19 @@ class Chromedriver extends events.EventEmitter {
     }
 
     if (this.adb) {
-      log.debug(`Cleaning any old adb forwarded port socket connections`);
+      const udidIndex = this.adb.executable.defaultArgs.findIndex((item) => item === '-s');
+      const udid = udidIndex > -1 ? this.adb.executable.defaultArgs[udidIndex + 1] : null;
+
+      if (udid) {
+        log.debug(`Cleaning this device's adb forwarded port socket connections: ${udid}`);
+      } else {
+        log.debug(`Cleaning any old adb forwarded port socket connections`);
```

Comment:
please describe what exactly the issue is and what this commit is trying to achieve. Here I see `Cleaning any old adb forwarded port`, but underneath no connections are cleaned if udid is `null`

Reply:
sorry, missed this condition.

Reply:
> please describe what exactly the issue is and what this commit is trying to achieve. Here I see `Cleaning any old adb forwarded port`, but underneath no connections are cleaned if udid is `null`

sorry, missed this condition.